pom.xml

```diff
@@ -171,6 +186,12 @@
       <artifactId>jersey-container-servlet</artifactId>
       <version>2.25</version>
     </dependency>
+	<dependency>
```

Comment:
There should be no explicit dependencies on repackaged libraries. They are intended internal only to other dependencies that applied it in a transitive manner.